source/ranges.tex

```diff
@@ -656,6 +657,29 @@
 type is integer-like.
 \end{note}
 
+\rSec2[range.prim.ssize]{\tcode{ranges::ssize}}
+
+\pnum
+The name \tcode{ssize} denotes
```

Comment:
I think this should be `The name \tcode{ranges::ssize} denotes` to be consistent with other CPO wording.

Reply:
I fixed this one, but there are several other cases which should be fixed as well - please open an edit issue for them:
```
The name \tcode{size} denotes a customization point
The name \tcode{empty} denotes a customization point
The name \tcode{data} denotes a customization point
The name \tcode{cdata} denotes a customization point
```

Reply:
#3752 fixes `size`, `empty`, and `data`; I've submitted #3793 to fix `cdata`.